### ISSUE TITLE
Jam session followup: Config files

### ISSUE THREAD

**Issue by ndokos** [distributed-system-analysis/pbench#2092]
We had to make several changes to the config files on staging in order to accommodate the pbench-server service.

The diff between the final config file on staging and what we have internally looks like this, somewhat edited to elide details:

```
--- pristine/pbench-server.cfg	2021-02-08 00:44:33.000000000 -0500
+++ ./pbench-server.cfg	2021-02-08 00:43:19.787412008 -0500
@@ -17,6 +17,12 @@
 roles = pbench-maintenance, pbench-prep, pbench-results, pbench-backup, pbench-sync-satellites
 environment = staging
 admin-email = admins@example.com
+commit_id = 0.71.0-1gf65d922f
+
+rest_log = %(pbench-logs-dir)s/pbench-server.log
+bycrypt_log_rounds = 5
+token_expiration_duration = 10
+
 
 [sosreports]
 user = vos
@@ -31,11 +37,19 @@
 secret_access_key = <some key>
 bucket_name = <some bucket>
 
+[Postgres]
+db_uri = postgres://user:password@127.0.0.1/<dbname>
+
 [Indexing]
-server = elasticsearch.example.com:NNNN
 index_prefix = staging-pbench
 bulk_action_count = 2000
 
+[elasticsearch]
+host = elasticsearch.example.com
+port = NNNN
+
+
+
 ###########################################################################
 # crontab roles
 
```

There is no difference between the installed pbench-server-default.cfg file and the one in the tree.

**Comment by ndokos** [comment]
@npalaska: I found uses of `token_expiration_duration` in the code, but no uses of `rest_log` or `bycrypt_log_rounds`, so I'm leaving those out of the config file altogether. Let me know if that is a mistake.

**Comment by ndokos** [comment]
Internal gerrit issue #237563 is dealing with this.

**Comment by ndokos** [comment]
Fixed by internal gerrit review #237563.

**Comment by portante** [comment]
Marked `invalid` as this is a change not tracked in this repo.
